core/src/lib.rs

```diff
@@ -38,11 +38,14 @@ pub struct EVMAddress(#[serde(with = "hex::serde")] pub [u8; 20]);
 /// Type alias for withdrawal payment, HashType is taproot script hash
 pub type WithdrawalPayment = (Txid, HashType);
 
-#[derive(Clone, Debug, Eq, Hash, PartialEq, PartialOrd, Ord, Serialize, Deserialize)]
+#[derive(
+    Clone, Debug, Eq, Hash, PartialEq, PartialOrd, Ord, Serialize, Deserialize, sqlx::FromRow,
+)]
 pub struct UTXO {
     pub outpoint: OutPoint,
     pub txout: bitcoin::TxOut,
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, sqlx::Type, sqlx::FromRow)]
```

Comment:
Ditto

Reply:
Here we are using it, is there any other way we can do this?

Reply:
You may use `(ByteArray66,)`. FromRow is always implemented automatically for any type `(T0,)` where `T0: sqlx::Type`.

Reply:
Done